plugins/flatpak/gs-flatpak.c

```diff
@@ -432,6 +432,11 @@ gs_flatpak_mark_apps_from_usb_remote (GsFlatpak *self,
 		 * so we add it to the GsApp here too */
 		gs_app_add_category (app, "USB");
 
+		/* if the category of the app is Blacklisted, remove it */
+                if (gs_app_has_category (app, "Blacklisted")) {
+			gs_app_remove_category (app, "Blacklisted");
+		}
+
```

Comment:
Again, I think it can all be done just from the EOS plugin. I am okay leaving this if the changes to the EOS plugin are too complex/error-prone. In which case you should also remove the curly brackets here (to make it more consistent with what we usually try to have as a coding style).